python/pyspark/errors/error_classes.py

```diff
@@ -39,6 +39,11 @@
       "Function `<func_name>` should return Column, got <return_type>."
     ]
   },
+  "JVM_ATTRIBUTE_NOT_SUPPORTED" : {
+    "message" : [
+      "Attribute `<attr_name>` is not supported in Spark Connect as it depends on the JVM. If you need to use this attribute, use the original PySpark instead of Spark Connect."
```

Comment:
Here, I'm not completely sure if "original" is the best term to use. Please let me know if someone have any better suggestions.